runtime/jvmti/jvmtiHeap10.c

```diff
@@ -668,17 +649,15 @@ wrapObjectIterationCallback(j9object_t *slotPtr, j9object_t referrer, void *user
 
 		/* Update the tag only if not autotagging or if it is a class, i.e., if the object is taggable */
 
-		if ( OBJECT_IS_TAGGABLE(iteratorData->env, vm, object) ) {
```

Comment:
`OBJECT_IS_TAGGABLE` also checked `J9VM_IS_INITIALIZED_HEAPCLASS_VM` - still required?

Reply:
The macro checked !autotag || initializedClass - since autotag is always false, all objects are taggable.

Reply:
"!a || b" doesn't even evaluate b if a is false.